releases/release-7.1.1.md

```diff
@@ -0,0 +1,180 @@
+---
+title: TiDB 7.1.1 Release Notes
+summary: Learn about the new features, compatibility changes, improvements, and bug fixes in TiDB 7.1.1.
+---
+
+# TiDB 7.1.1 Release Notes
+
+Release date: July xx, 2023
+
+TiDB version: 7.1.1
+
+Quick access: [Quick start](https://docs.pingcap.com/tidb/v7.1/quick-start-with-tidb) | [Production deployment](https://docs.pingcap.com/tidb/v7.1/production-deployment-using-tiup) | [Installation packages](https://www.pingcap.com/download/?version=v7.1.1#version-list)
+
+## Compatibility changes
+<!--1 tw:@Oreoxmt-->
+- TiKV changes the default value of [<code>rocksdb.\[defaultcf\|writecf\|lockcf\].compaction-guard-min-output-file-size</code>](/tikv-configuration-file.md#compaction-guard-min-output-file-size) from `"8MB"` to `"1MB"` to reduce the data volume of compaction tasks in RocksDB [#14888](https://github.com/tikv/tikv/issues/14888) @[tonyxuqqi](https://github.com/tonyxuqqi)
+- TiDB introduces a new system variable `tidb_lock_unchanged_keys` to control whether to lock unchanged keys [#44714](https://github.com/pingcap/tidb/issues/44714) @[ekexium](https://github.com/ekexium)
+
+## Improvements
+
++ TiDB
+    <!--2 tw:@Oreoxmt-->
+    - Plan Cache supports queries with more than 200 parameters [#44823](https://github.com/pingcap/tidb/issues/44823) @[qw4990](https://github.com/qw4990)
+    - Optimize the performance of reading chunks related to disk [#45125](https://github.com/pingcap/tidb/issues/45125) @[YangKeao](https://github.com/YangKeao)
+    (dup: release-7.2.0.md > Improvements> TiDB)- Optimize the logic of constructing index scan range so that it supports converting complex conditions into index scan range [#41572](https://github.com/pingcap/tidb/issues/41572) [#44389](https://github.com/pingcap/tidb/issues/44389) @[xuyifangreeneyes](https://github.com/xuyifangreeneyes) @[xuyifangreeneyes](https://github.com/xuyifangreeneyes)
+    (dup: release-7.2.0.md > Improvements> TiDB)- When the retry leader of stale read encounters a lock, TiDB forcibly retries with the leader after resolving the lock, which avoids unnecessary overhead [#43659](https://github.com/pingcap/tidb/issues/43659) @[you06](https://github.com/you06) @[you06](https://github.com/you06)
```

Comment:
```suggestion
    (dup: release-7.2.0.md > Improvements> TiDB)- Optimize the logic of constructing index scan range so that it supports converting complex conditions into index scan range [#41572](https://github.com/pingcap/tidb/issues/41572) [#44389](https://github.com/pingcap/tidb/issues/44389) @[xuyifangreeneyes](https://github.com/xuyifangreeneyes)
    (dup: release-7.2.0.md > Improvements> TiDB)- When the retry leader of stale read encounters a lock, TiDB forcibly retries with the leader after resolving the lock, which avoids unnecessary overhead [#43659](https://github.com/pingcap/tidb/issues/43659) @[you06](https://github.com/you06)
```